webxr/openxr/mod.rs

```diff
@@ -498,7 +499,7 @@ fn init_device_for_adapter(
             adapter.as_raw(),
             D3D_DRIVER_TYPE_UNKNOWN,
             ptr::null_mut(),
-            d3d11::D3D11_CREATE_DEVICE_BGRA_SUPPORT,
+            d3d11::D3D11_CREATE_DEVICE_BGRA_SUPPORT | d3d11::D3D11_CREATE_DEVICE_DEBUG,
```

Comment:
Should we remove this, or at least file an issue so we don't end up shipping with it on? It might even be nice to make it configurable from the embedding.

Reply:
I'll remove it for now